sdcm/nemesis.py

```diff
@@ -3550,12 +3552,14 @@ def disrupt_memory_stress(self):
         else:
             raise UnsupportedNemesis(f"{self.target_node.distro} OS not supported!")
         self.log.info('Try to allocate 120% available memory, the allocated memory will be swaped out')
-        self.target_node.remoter.run(
-            "stress-ng --vm-bytes $(awk '/MemAvailable/{printf \"%d\\n\", $2 * 1.2;}' < /proc/meminfo)k --vm-keep -m 1 -t 100")
 
-        self.log.info('Try to allocate 90% total memory, the allocated memory will be swaped out')
-        self.target_node.remoter.run(
-            "stress-ng --vm-bytes $(awk '/MemTotal/{printf \"%d\\n\", $2 * 0.9;}' < /proc/meminfo)k --vm-keep -m 1 -t 100")
+        with ignore_reactor_stall_errors():
```

Comment:
this line should now be removed, once the decorator is in place

Reply:
ping @KnifeyMoloko

Reply:
Fixed